README.md

```diff
@@ -127,13 +156,15 @@ limitd.take(type, key, [count], (err, result) => {
 -  `key`: the identifier of the bucket.
 -  `count`: the amount of tokens you need. This is optional and the default is 1.
 -  `configOverride`: caller-provided bucket configuration for this operation
+-  `erlIsActiveKey`: (string) the identifier of the ERL activation for the bucket. Must be passed for buckets that have ERL configured.
```

Comment:
This is shown as `erlActiveKey` (missing `Is`) in line 148. Make sure both are in sync.